cli/cmd/install.go

```diff
@@ -187,6 +195,22 @@ func installOdigos(ctx context.Context, client *kube.Client, ns string, config *
 
 }
 
+func parseNodeSelectorFlag() (map[string]string, error) {
+	nodeSelector := make(map[string]string)
+	if len(nodeSelector) == 0 {
+		return nodeSelector, nil
+	}
```

Comment:
this is creating an empty map, and check if that map is empty

```suggestion
	nodeSelector := make(map[string]string)
	if len(nodeSelectorFlag) == 0 {
		return nodeSelector, nil
	}
```